bedrock/settings/base.py

```diff
@@ -101,6 +101,9 @@ def path(*args):
 # http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGE_CODE = 'en-US'
 
+# Use Ship It as the source for product_details
+PROD_DETAILS_URL = 'https://product-details.mozilla.org/1.0/'
```

Comment:
Could you make this configurable? It'd be nice to be able to switch back to SVN if we have to w/o having to deploy.

```python
PROD_DETAILS_URL = config('PROD_DETAILS_URL', default='https://product-details.mozilla.org/1.0/')
```